Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/customize-image.md

## Add Environment Variables Locally

> The Astro CLI comes with the ability to  bring in Environment Variables from a specified file by running `$ astro dev start` with an `--env` flag as seen below:

Reviewer: ```suggestion
To import Environment Variables from a specific file, run `$ astro dev start` with an `--env` flag:
```
